### DISCUSSION TITLE
MikTeX silent/hidden install switches

### DISCUSSION THREAD

**Discussion by Emohr77** [MiKTeX/miktex#1069] [Q&A]
Working on packaging this for SCCM delivery but have not found the silent/hidden switches.  I have tried  --quiet  --unattended and the autoinstall=yes  and a combination of those and the install progress screen always pops up. Anyone get this to install hidden/quiet without prompts or interface?  

Thanks

**Comment by MuhKuh7** [comment]
Try ` --unattended --auto-install=yes`
